src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings

```diff
@@ -576,6 +576,30 @@ oben genannten Länder. Wenn Sie zusätzlich Angaben zum Beginn Ihrer Symptome m
 
 "ExposureSubmissionIntroduction_ListItem4" = "Wenn Sie keine TAN haben, können Sie diese telefonisch anfragen.";
 
+/* Exposure Submission Testresult available*/
+"ExposureSubmissionTestresultAvailable_Title" = "Ihr Testergebnis liegt vor";
+
+"ExposureSubmissionTestresultAvailable_AccImageDescription" = "Ein Smartphone übermittelt einen positiven Testbefund verschlüsselt ins System.";
+
+"ExposureSubmissionTestresultAvailable_Consent_granted" = "Einverständnis\n“Andere warnen” erteilt";
```

Comment:
Hi, maybe this is because of the leading linebreak but the leading quotation marks should be at the bottom
`„Andere warnen“`